faqs/different_versions.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Differences between CC, CC:Tweaked, OC, and OC2
+search: differences different
+---
+Over the years, multiple different computer mods have been created. Each of these mods have their own unique features, and cater to different play styles. Below is a brief overview of the differences between the most popular computer mods.
+
+## Brief Descriptions
+
+**CC (ComputerCraft)**
+
+ComputerCraft is the original version of ComputerCraft, developed by Dan200. It's available for Minecraft 1.6.4 up to 1.12.2. This version is missing many of the modern features that CC:Tweaked contains. The computers are programmed using Lua 5.1.
+
+**CC:Tweaked**
+
+CC:Tweaked is an updated version of ComputerCraft, maintained by SquidDev and many members of the community. It adds many new features on top of the original ComputerCraft, including websocket support, new peripherals, and DFPWM audio support. CC:Tweaked operates on the value of backwards compatibility, attempting to ensure that as many existing programs as possible continue to work without modification. Newer versions of the mod also include support for Lua 5.2.
+
+**OC (OpenComputers)**
+
+OpenComputers is a separate mod from ComputerCraft, developed by Asie. It focuses on a more 'realistic' approach to computers within Minecraft, featuring more complex (but limited) hardware components, and its own set of peripherals. OpenComputers is available for Minecraft 1.6.4 up to 1.12.2, and is also programmed using Lua.
+
+**OC2 (OpenComputers 2)**
+
+OC2 is the successor to OpenComputers, though make no mistake: it's not just a continuation like CC:Tweaked is. OC2 introduces the RISC-V architecture, which is a significant departure from the original OpenComputers mod. This mod is still in development, and aims to bring modern features and improvements to the OpenComputers experience. OC2 is available for Minecraft 1.18.1 and 1.18.2. OC2's default operating system is Linux, and Lua is included to enable easy scripting.
+
+## Main Differences
+
+If you're looking for a bullet-point list of differences between the mods, here you go:
+
+- **Programming Language**: All four mods use Lua, but OC2 uses RISC-V, meaning it can use anything else you desire as well.
+- **Operating System**
+  - **CC/CC:Tweaked**:
+    - CraftOS: Simplifies scripting for users as it provides a large set of APIs available in the global environment.
+  - **OpenComputers**:
+    - OpenOS: A more modular operating system that requires the user to explicitly import non-standard libraries.
```

Comment:
haven't used craftos much but saying that one OS has everything loaded in the global env and in the other you have to import the libraries explicitly is a cosmetic technicality IMO. I'd rather point out that OpenOS has a different directory structure that tries to mimic UNIX-likes, with `/home`, `/bin`, `/usr` etc. and you're free to edit this while iirc CraftOS just had a ROM directory, and that was non-editable(?).

Also maybe it would be good to point out that some APIs of openos will be familiar to craftos users.